eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```
@@ -285,5 +285,11 @@ services:
     ezpublish.image_alias.variation_path_generator.alias_directory:
         class: eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPathGenerator\AliasDirectoryVariationPathGenerator
 
+    eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPathGenerator\WebpFormatVariationPathGenerator:
+        decorates: ezpublish.image_alias.variation_path_generator
+        arguments:
+            $innerVariationPathGenerator: '@.inner'
+            $filterConfiguration: "@liip_imagine.filter.configuration"
```

CS: quoting inconsistency
```suggestion
            $filterConfiguration: '@liip_imagine.filter.configuration'
```